config.example.yaml

```diff
@@ -0,0 +1,11 @@
+bot-token: bot_token_goes_here
+owners: # user IDs of bot owners with admin perms
+  - 1234
+  - 5678
+
+# emoji that should be used for karma
+# either a unicode emoji, name of reaction, or emoji ID
+# mapped to karma value
+emoji:
+  😊: 1
+  1000161402704633956: -5
```

Comment:
What is this

Reply:
The config file has a mapping of emoji to value
emoji can be either a literal emoji, the discord emoji ID, or the name of the emoji

Reply:
Ok add comment for what emoji like :smile:

Reply:
The :emoji:

Reply:
wdym

Reply:
What server emoji from and what emoji called like # :sus: Uni of ww discord

Reply:
oh nah these are just examples they're overwritten in the actual config, not meant to have actual emoji. That ID isnt even an emoji im pretty sure

Reply:
👌😳